test/transforms/subsetFonts.js

```diff
@@ -2771,9 +2773,9 @@ describe('transforms/subsetFonts', function() {
           expect(warnSpy, 'to have calls satisfying', function() {
             warnSpy({
               message: expect
-                .it('to contain', 'OpenSans.ttf is missing these characters')
-                .and('to contain', 'U+4E2D (中)')
-                .and('to contain', 'U+56FD (国)')
+                .it('to contain', 'Missing glyph fallback detected')
+                .and('to contain', '\\u{4e2d} (中)')
+                .and('to contain', '\\u{56fd} (国)')
```

Comment:
Why is the `\u{nnnn}` syntax better?

Reply:
Because Mathias says so

![img_20180601_210131](https://user-images.githubusercontent.com/331790/40858608-1dbd272e-65df-11e8-8a02-137a4b1bdebd.jpg)

Reply:
Weak :)

Reply:
I know. But the timing was perfect :)

This is the article where he outlines the unicode escape sequence problems: https://mathiasbynens.be/notes/javascript-unicode#escape-sequences

This syntax lets you describe astral planes as well instead of having to create surrogate planes

Reply:
Okay, I'm buying that :)

Next stop --- the stars 🌃